ting_client_class.inc

```diff
@@ -76,10 +78,12 @@ class ting_client_class {
     if (isset($settings['custom_parse'])) {
       if (method_exists($request, $settings['custom_parse'])) {
         $function = $settings['custom_parse'];
-        return $request->$function($result);
+        $result = $request->$function($result);
       }
     }
 
+    // Cache result in ting_mockup.
+    module_invoke_all('ting_client_mockup_cache_set', $request->cacheKey(), $result);
```

Comment:
For mig ser det ikke ud som om du tjekker efter fejl?
Hvad nu hvis $result indeholder et "error" svar, så skal det vel ikke caches?